ArduinoController/ArduinoController.ino

```diff
@@ -10,43 +10,41 @@
 #include "StartController.h"
 #include "PinJSON.h"
 
-const uint8_t KILLPIN = 50; // Needs to remain global (therefore outside of JSON) for lambda interrupt
-const uint8_t KILL_ADDR = 10;
 const uint32_t CONTROLLER_CNT = 13u;
 class IController* controllers[CONTROLLER_CNT];
 
 void setup() {
   Serial.begin(115200);
-  Serial.print('R');
-  DynamicJsonBuffer buffer;
-  auto json = buffer.parse(PinJSON::json);
```

Comment:
I knew this day would come when I would need to explain my self... In fact, I prepared by writing a several page exploratory research paper of the pros and cons of using JSON with microcontrollers and serial communication.

In summary, serial UART communication is not near as fast as traditional USB2.0 as one might think. To demonstrate, some JSON strings could take up to 250 ms to transfer with serial UART when it would only take 2.9 ms via USB2.0. (That's 86 times slower)

While this move appears to be a step backward, I actually took this goal into consideration when writing the serialization subroutines. The two base functions `readData` and write `writeData` don't just serialize primitive types.

As the test, `Test_StructDataType`, demonstrates, user-defined structs may also be serialized/deserialized between devices. With this, a simple pin configuration struct can be created for both sides (which is required anyway) and then pins can be configured in one blazing fast call.